docker/client/container.sls

```diff
@@ -6,12 +6,34 @@ include:
 {%- for name, container in client.get('container', {}).iteritems() %}
   {%- set id = name %}
   {%- set required_containers = [] %}
+{#
+get_docker_py:
```

Comment:
No way, there are salt resources to work with pip anyway you want to make it possible to choose install method (pkg, pip, etc.) and prefer pkg where possible (eg. see installation of docker-compose)